pkg/tracing/datadog/logger.go

```diff
@@ -0,0 +1,15 @@
+package datadog
+
+import "github.com/traefik/traefik/v2/pkg/log"
+
+type DatadogLogger struct {
+	logger log.Logger
+}
+
+func newDatadogLogger(logger log.Logger) *DatadogLogger {
+	return &DatadogLogger{logger: logger}
+}
+
+func (d DatadogLogger) Log(msg string) {
+	d.logger.Debug(msg)
+}
```

Comment:
Why force `Debug` level on Log() ?
It seems to be explicitly [supported by zerolog](https://github.com/rs/zerolog#logging-without-level-or-message)

Reply:
@mloiseleur yes I agree with you, only took it as it's implemented the same way in branch v3.0, so took it for consistency. Should I remove it ?

Reply:
@mloiseleur Actually in v2.10 the logger used is not zerolog , its "github.com/traefik/traefik/v2/pkg/log" and does not have any such general log function.

Reply:
ah, that was introduced in v3, that's right !